nextcloudappstore/core/api/v1/release/info.xsd

```diff
@@ -245,7 +245,7 @@
     <!-- first level elements -->
     <xs:simpleType name="id">
         <xs:restriction base="limited-string">
-            <xs:pattern value="[a-z]+[a-z_]*[a-z]+"/>
+            <xs:pattern value="[a-z]+[a-z0-9_]*[a-z0-9]+"/>
```

Comment:
I would suggest something like `[a-z]_?([a-z0-9]+_?)*[a-z0-9]+`
This would prohibit name like  `a________________________1`